### PR TITLE
feat!: make `chainID` configurable for `TestBaseConfig`

### PR DESCRIPTION
**Pull request by rootulp**
Closes https://github.com/celestiaorg/celestia-core/issues/971

### CODE REVIEW

**Review by evan-forbes — COMMENTED** [review]
LGTM but will defer to @renaynay

**Review by evan-forbes — DISMISSED** [review]

File: config/config.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func DefaultConfig() *Config {
 // TestConfig returns a configuration that can be used for testing
 func TestConfig() *Config {
 	return &Config{
-		BaseConfig:      TestBaseConfig(),
+		BaseConfig:      TestBaseConfig("tendermint_test"),
```
[the remark this answers is not in the document]
Comment (rootulp) [reply config/config.go]: resolved by https://github.com/celestiaorg/celestia-core/pull/972/commits/3b30796d66d59e0bae48cc8e51be1cf9b59437ad

**Review by renaynay — APPROVED** [review]

**Comment by renaynay** [comment]
Thank you 🙏🏻

**Comment by renaynay** [comment]
FWIW @rootulp @evan-forbes  - we could also pass a chainID inside `testnode.New` instead of this solution so that we can PR to app instead of core. I am happy to do that!

**Comment by evan-forbes** [comment]
that's makes the most sense to me, since we should try to always test with celestia-app and not just tendermint

**Comment by rootulp** [comment]
Closed in favor of https://github.com/celestiaorg/celestia-app/pull/1445

**Pull request closed without merging**